requirements/requirements.in

```diff
@@ -43,6 +43,8 @@ click
 # torch at 1.13.0 or later gives trouble with cuda dependencies
 torch<1.13.0
 torchvision==0.13.1
+pytorch-lightning
+ray-lightning
```

Comment:
No, because whenever we use a `sematic_pipeline` target, the dependencies need to be available in our bazel workspace. The only time examples can get away with dependencies *only* in the example's `requirements.txt`/`sematic_example` target are when they are local only.